#Request/Reply for Image Service

#Request by indicating the number of images to be captured
uint32 num_images

---

#Reply with the size of the captured image
uint32 width
uint32 height